torchinductor/compile_fx.py

```diff
@@ -231,6 +232,11 @@ def is_not_gradout(x):
 
 def compile_fx(model_: torch.fx.GraphModule, example_inputs_: List[torch.Tensor]):
     """Main entrypoint to a compile given FX graph"""
+
+    if not is_aot_autograd_safe_to_run(model_, example_inputs_):
+        log.debug("Aot Autograd is not safe to run, so falling back to eager")
```

Comment:
Let's make this a `log.warning`, this seems worse than the graph break warning we already print out.